### DISCUSSION TITLE
Does NNI Supports symmetric quantization?

### DISCUSSION THREAD

**Discussion by twmht** [microsoft/nni#4169] [Q&A]
According to the TensorRT document (https://docs.nvidia.com/deeplearning/tensorrt/archives/tensorrt-700/tensorrt-developer-guide/index.html#set_tensor_mp_c), symmetric quantization is supported.

Therefore when using TensorRT, we would see the warnings

```
DynamicRange(min: -0.424211, max: 2.82148). Dynamic range should be symmetric for better accuracy)
```

But NNI only supports asymmetric  quantization.

Any idea?

**Comment by linbinskn** [comment]
We have planned to support different dtypes and qschemes in this [pr](https://github.com/microsoft/nni/pull/4137) which includes symmetric quantization.

**Reply by twmht** [reply]
nice. hopefully this would be merge.

**Reply by chenbohua3** [reply]
working on it :)
